pagopa/io-app
# feat: [IOCOM-1029] Add support for multi image in the detail of a message

## Short description
This PR adds support for multi image in the detail of a message. The component first tries to retrieve the logo based on the `service_id`, if it is not available then it tries to retrieve the logo based on the `organization_fiscal_code`

## List of changes proposed in this pull request
- updated the `logoUri` property of the `OrganizationHeader` component

## How to test
Using `io-dev-api-server`, generate a message. Activate `experimental ds` and open a message. Check that the logo is displayed correctly.
